### PR TITLE
Update DAG's for using own DB user and schema

### PR DESCRIPTION
**Pull request by cvriel**
Updating DAGs (covid19,overlastgebieden,precariobel,reclamebel) to cope with AAD user database and related datbase sketch schema

### CODE REVIEW

File: src/dags/covid19.py [hunk 1]
```diff
@@ -60,47 +66,60 @@
...
     ]
 
-    # 6. Create TABLE
-    create_tables = [
-        BashOperator(
-            task_id=f"create_table_{key}",
-            bash_command=f"psql {pg_params()} < {tmp_dir}/{key}.sql",
+    # 5. Dummy operator acts as an interface between parallel tasks to
+    # another parallel tasks with different number of lanes
+    # (without this intermediar, Airflow will give an error)
```
Comment (MaguellSandifort) [inline src/dags/covid19.py]: Nitpick:
```suggestion
    # (without this intermediary, Airflow will give an error)
```

File: src/dags/overlastgebieden.py [hunk 2]
```diff
@@ -101,20 +114,23 @@
         prefix_table_name=f"{DAG_ID}_",
         postfix_table_name="_new",
         rename_indexes=False,
-        pg_schema="public",
+        pg_schema=tmp_database_schema,
     )
 
     # 7. Revalidate invalid geometry records
```
Comment (MaguellSandifort) [inline src/dags/overlastgebieden.py:120]: Should these be removed? Looks out of order

File: src/dags/precariobelasting.py [hunk 3]
```diff
@@ -110,131 +134,132 @@ def clean_data(file_name: str) -> None:
...
+                },
                 pass_value=1,
             )
         )
 
         total_checks = count_checks + geo_checks
         check_name[file_name] = total_checks
 
-    # 8. Execute bundled checks (step 7) on database
+    # 9. Execute bundled checks (step 7) on database
```
Comment (MaguellSandifort) [inline src/dags/precariobelasting.py]:
```suggestion
    # 9. Execute bundled checks (step 8) on database
```

File: src/dags/precariobelasting.py [hunk 4]
```diff
@@ -110,131 +134,132 @@ def clean_data(file_name: str) -> None:
...
-        PostgresTableRenameOperator(
-            task_id=f"rename_table_{file_name}",
-            old_table_name=file_name,
-            new_table_name=f"{dag_id}_{file_name}",
-        )
-        for file_name in data_endpoints.keys()
-    ]
-
-    # 13. Add column TITLE as its was set in the display property in the metadataschema
+    # 11. Add column TITLE as its was set in the display property in the metadataschema
```
Comment (MaguellSandifort) [inline src/dags/precariobelasting.py]: Nitpick:
```suggestion
    # 11. Add column TITLE as it was set in the display property in the metadataschema
```

**Review by MaguellSandifort — COMMENTED** [review]
Some minor nitpicks, otherwise LGTM!

File: src/dags/overlastgebieden.py (hunk 2 above)
Comment (cvriel) [reply src/dags/overlastgebieden.py:120]: Yes definitely, it should be removed

**Pull request merged**